crashlogging/src/main/java/com/automattic/android/tracks/crashlogging/CrashLoggingUser.kt

```diff
@@ -1,7 +1,7 @@
 package com.automattic.android.tracks.crashlogging
 
 data class CrashLoggingUser(
```

Comment:
I'm guessing you can have only one or two but not all properties, right?
If this was "all or nothing", you could also make `CrashLoggingUser?`, but I assume some clients will have only part of the properties.

Reply:
Yes, exactly. E.g. in case of Pocket Casts, users only have mails. I'd like to make this API flexible for this.